packages/ns-api/files/ns.reverseproxy

```diff
@@ -428,18 +294,11 @@ elif cmd == 'call':
                 raise ValidationError('description', 'required')
             if 'allow' in data and not isinstance(data['allow'], list):
                 raise ValidationError('allow', 'invalid', data['allow'])
+
+            # normalize optional parameters
+            allow = data['destination'] if 'allow' in data else []
```

Comment:
Why setting 'destination' as allow?
I think this one should be:
```
allow = data.get('allow', [])
```

Same on line 319.

Reply:
Definetly a typo, most fun part is that the migration is still successful (?)
![image](https://github.com/NethServer/nethsecurity/assets/8052641/d52dad26-4175-49f2-aacf-7e302ede5cb4)
 
:shrug: